security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,26 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-52"]
+== OpenShift Compliance Operator 0.1.52
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.52:
+
+* link:https://access.redhat.com/errata/RHBA-2022:4657[RHBA-2022:4657 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-52-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `OpenScap` container would crash due to a mount permission issue in a security environment where `DAC_OVERRIDE` capability is dropped. Now, executable mount permissions are applied to all users. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082151[*BZ#2082151*])
+
+* Previously, the compliance rule `ocp4-configure-network-policies` could be configured as `MANUAL`. Now, compliance rule `ocp4-configure-network-policies` is set to automatic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072431[*BZ#2072431*])
+
+* Previously, the Cluster Autoscaler would fail to scale down due to the Compliance Operator scan pods were never removed after a scan. Now, the pods are removed from each node by default unless explicitly saved for debugging purposes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075029[*BZ#2075029*])
```

Comment:
I feel like I'm missing a "because" in the first sentence somewhere. Perhaps the following?

> Previously, the Cluster Autoscaler would fail to scale down because the Compliance Operator scan pods were never removed after a scan.